pylint/extensions/check_mccabe.py

```diff
@@ -0,0 +1,176 @@
+# -*- coding: utf-8 -*-
```

Comment:
You're using somewhere non-ascii characters? This encoding cookie might be superfluous.